### ISSUE TITLE
Settings selections hard to see in "Light" Theme (Win10 Chrome & Chromium)

### ISSUE THREAD

**Issue by mkmori** [processing/p5.js-web-editor#2007]
### How would this new feature help [increase access](https://github.com/processing/p5.js/blob/main/contributor_docs/access.md) to the p5.js Editor?
Suggestion to improve usability of Settings menu in "light" theme.

#### Feature enhancement details:
In my Chrome & Chromium (Vivaldi) browsers esp. (Win10), the weight, color, and lack of other indicators make it difficult to see what settings are selected in "Light" mode (personally). (It looks about the same in Safari Mobile, but I understand the mobile editor isn't here yet.)
| Chrome/Chromium: | Slightly better in Firefox: |
|---|---|
| ![image](https://user-images.githubusercontent.com/4692695/161436409-4b649aca-8178-4e0b-9ddf-fec29671ecaf.png) | ![image](https://user-images.githubusercontent.com/4692695/161436341-c0c4804f-4e22-43e7-ae44-6e770831de1e.png) |

(I thought I saw this mentioned somewhere, but I can't find it now--maybe it was in Discourse...?)

**Comment by 3ru** [comment]
I felt this too! I will create a PR soon!
